ui/packages/consul-ui/app/components/topology-metrics/index.hbs

```diff
@@ -17,7 +17,11 @@
     </div>
   {{#each @topology.Downstreams as |item|}}
     <TopologyMetrics::Card
+<<<<<<< HEAD
       @item={{item}}
+=======
+      @items={{@topology.Downstreams}}
+>>>>>>> 56c2ff56e (ui: Update to not return metrics for ingress gateways (#9081))
```

Comment:
`as |item|` in the loop appears to be completely unused if taking this change, not sure what's intended